application/modules/admin/models/logging_model.php

```diff
@@ -32,6 +32,9 @@ public function findLogs($search = "", $module = "")
 				$userId = $search;
 			}
 		}
+		
+		// prevent sql injection
+		$module = mysqli_real_escape_string($this->db->conn_id, $module);
```

Comment:
Here use `$this->db->escape()` instead of the mysqli function, since it's made with CI I'd like to keep this consistency

Reply:
Example if $module is: Register
$this->db->escape() would transform it into: 'Register'
With the ' it's going to create a sql query error

Reply:
alright - `escape_str` would be more suitable